CONTRIBUTING.md

```diff
@@ -1,64 +1,76 @@
 
-```
-cd openzeppelin-contracts
-git remote add upstream https://github.com/OpenZeppelin/openzeppelin-contracts.git
-git fetch upstream
-git pull --rebase upstream master
-```
-NOTE: The directory `openzeppelin-contracts` represents your fork's local copy.
+- The issue is not a duplicate issue.
+- The issue has not been fixed in the latest release of Klaytn Contracts.
+Please do not use the issue tracker for personal support requests. Use developer@klaytn.foundation for the personal support requests.
 
-2) Branch out from `master` into `fix/some-bug-#123`:
-(Postfixing #123 will associate your PR with the issue #123 and make everyone's life easier =D)
-```
-git checkout -b fix/some-bug-#123
-```
+When you report a bug, please make sure that your report has the following information.
+- Steps to reproduce the issue.
+- A clear and complete description of the issue.
+- Code and/or screen captures are highly recommended.
 
-3) Make your changes, add your files, commit, and push to your fork.
+After confirming your report meets the above criteria, [submit the issue](https://github.com/klaytn/klaytn-contracts/issues). Please use [labels](#usage-of-labels) to categorize your issue.
 
-```
-git add SomeFile.js
-git commit "Fix some bug #123"
-git push origin fix/some-bug-#123
-```
+### Feature Requests
 
-4) Run tests, linter, etc. This can be done by running local continuous integration and make sure it passes.
+You can also use the [issue tracker](https://github.com/klaytn/klaytn-contracts/issues) to request a new feature or enhancement. Note that any code contribution without an issue link will not be accepted. 
 
-```bash
-npm test
-npm run lint
-```
+Please submit an issue explaining your proposal first so that the Klaytn developer community can fully understand and discuss the idea. Please use [labels](#usage-of-labels) for your feature request as well.
 
-5) Go to [github.com/OpenZeppelin/openzeppelin-contracts](https://github.com/OpenZeppelin/openzeppelin-contracts) in your web browser and issue a new pull request.
+#### Usage of Labels
 
-*IMPORTANT* Read the PR template very carefully and make sure to follow all the instructions. These instructions
-refer to some very important conditions that your PR must meet in order to be accepted, such as making sure that all tests pass, JS linting tests pass, Solidity linting tests pass, etc.
+You can use the following labels:
 
-6) Maintainers will review your code and possibly ask for changes before your code is pulled in to the main repository. We'll check that all tests pass, review the coding style, and check for general code correctness. If everything is OK, we'll merge your pull request and your code will be part of OpenZeppelin Contracts.
+Labels for initial issue categories:
 
-*IMPORTANT* Please pay attention to the maintainer's feedback, since it's a necessary step to keep up with the standards OpenZeppelin Contracts attains to.
+- issue/bug: Issues with the code-level bugs.
```

Comment:
Issues types should reflect:
[Issues and Bugs]
[Feature Requests]
[Code Contribution]

as we listed above thanks

Reply:
refactored